pyron-plugin-impl/src/main/scala/com/cloudentity/pyron/plugin/impl/transformer/TransformRequestPlugin.scala

```diff
@@ -170,7 +190,7 @@ object TransformRequestOpenApiConverter extends OpenApiPluginUtils {
                 }.toList
 
             pathParamsToTransform.foreach { case (swaggerParam, valueOrRef) =>
-              valueOrRef match {
+              (valueOrRef: @unchecked) match { // no default case, fail fast
```

Comment:
Why this change? Can it throw exception now? Let's avoid throwing exception in our code.

Reply:
it could throw exception before, and it still can throw it now, @unchecked makes it explicit
I think it's a fail-fast approach, we already discussed in regards of reading the config, hence the comment.

Reply:
In what circumstances can it throw exception?

Reply:
Oh, my bad, it was a bug. You are right, it wouldn't be compiler error. Could you please change the pattern matches over ValueOrRef from flat one like this to nested, i.e.:
```
valueOrRef match {
  case Value(v) => ...
  case ref: RefType =>
     ref match {
         AuthnRef(_) => ...
         BodyRef(_) => ...
     }
}
```
? This way compiler will guide the developer when adding new ref type as there would be non-exhaustive pattern matching errors.

Reply:
Oops, sorry for confusion, IntelliJ tricked me when I added new RefType and did not fail compilation. After second build it failed. Actually, there is no need to rework the pattern matching to fail at non-exhaustive pattern matching, it would fail anyway, or it wouldn't? :)

Reply:
reverted to the old ways